ckanext/cloudstorage/logic/action/multipart.py

```diff
@@ -96,6 +96,8 @@ def initiate_multipart(context, data_dict):
     uploader = ResourceCloudStorage({'multipart_name': name})
     res_name = uploader.path_from_filename(id, name)
 
+    log.info('initiate_multipart %s upload started' % (res_name))
```

Comment:
Instead of logging the resource's name we should use its id. The reason is that resource names are not unique.

Reply:
Yeah, I had the same concern, but the res_name variable that I'm using here comes from the previous line, which uses the resource id:

```
def path_from_filename(self, rid, filename):
        """
        Returns a bucket path for the given resource_id and filename.
        return os.path.join(
            self.parent_directory_name, 'resources',
            rid,
            munge.munge_filename(filename)
        )
```

Reply:
Several can resource ids can resolve into the same name (i.e. CSV). In that case we wouldn't be able to determine which resource has an error. With the resource id we can look it up with resource_show API